File: multitool-otel-setup-workspace/iteration-1/eval-2-python-new-user/with_skill/outputs/tracing.py
import os
from opentelemetry.sdk.trace import TracerProvider
from opentelemetry.sdk.trace.export import BatchSpanProcessor
from opentelemetry.sdk.resources import Resource, SERVICE_NAME, SERVICE_VERSION
from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
from opentelemetry import trace

resource = Resource.create({
    SERVICE_NAME: "order-service",
    SERVICE_VERSION: os.environ.get("APP_VERSION", ""),
})

exporter = OTLPSpanExporter(
    endpoint="https://api.multitool.run/otlp/v1/traces",
    headers={"api-key": os.environ.get("MULTI_API_KEY", "")},
)

provider = TracerProvider(resource=resource)
provider.add_span_processor(BatchSpanProcessor(exporter))
trace.set_tracer_provider(provider)

File: multitool-otel-setup-workspace/iteration-1/eval-3-troubleshooting/without_skill/inputs/tracing.ts
import { NodeSDK } from '@opentelemetry/sdk-node'
import { resourceFromAttributes } from '@opentelemetry/resources'
import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions'
import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'

const traceExporter = new OTLPTraceExporter({
  url: 'https://api.multitool.run/otlp/v1/traces',
  headers: {
    'api-key': process.env['MULTI_API_KEY'] ?? '',
  },
})

const sdk = new NodeSDK({
  resource: resourceFromAttributes({
    [ATTR_SERVICE_NAME]: 'user-service',
    [ATTR_SERVICE_VERSION]: process.env['APP_VERSION'] ?? 'unknown',
  }),
  traceExporter,
})

sdk.start()
